src/ngx_lua_resty_lmdb_module.c

```diff
@@ -423,3 +423,37 @@ lmdb_encrypt_func(const MDB_val *src, MDB_val *dst, const MDB_val *key, int encd
 }
 
 
+int ngx_lua_resty_lmdb_ffi_env_info(ngx_lua_resty_lmdb_ffi_status_t *lst, const char **err)
+{
+    ngx_lua_resty_lmdb_conf_t      *lcf;
+    MDB_stat                        mst;
+    MDB_envinfo                     mei;
+
+    lcf = (ngx_lua_resty_lmdb_conf_t *) ngx_get_conf(ngx_cycle->conf_ctx,
+                                                     ngx_lua_resty_lmdb_module);
+
+    if (lcf == NULL || lcf->env == NULL) {
+        *err = "no LMDB environment defined";
+        return NGX_ERROR;
+    }
+
+    if (mdb_env_stat(lcf->env, &mst)) {
+        *err = "mdb_env_stat failed";
+        return NGX_ERROR;
+    }
+
+    if (mdb_env_info(lcf->env, &mei)) {
+        *err = "mdb_env_info failed";
+        return NGX_ERROR;
+    }
+
+    lst->map_size = mei.me_mapsize;
+    lst->page_size = mst.ms_psize;
+    lst->max_map_size = mei.me_mapsize;
+    lst->used_pages = mei.me_last_pgno + 1;
```

Comment:
Is this calculation accurate? I can not derive directly from the LMDB doc that `mei.me_last_pgno + 1` equals to used pages. Could you explain?

Reply:
`used_pages` is not mean truly used pages, I think this situation is more likely `alloc_ pages`.

Reply:
cc @dndx

Reply:
We should use a different name in this case, maybe just call it `last_used_page` instead. 

`used_pages` is confusing as it suggests this can be used to estimate hjow many pages are actually occupied, and this is not what it is.